core/src/apps/homescreen/device_menu.py

```diff
@@ -24,11 +24,21 @@ class SubmenuId:
     POWER = const(8)
 
 
-def _get_hostname(ble_addr: bytes, hostname_map: dict[bytes, str]) -> str:
-    if (hostname := hostname_map.get(ble_addr)) is None:
+def _get_hostinfo(
+    ble_addr: bytes, hostname_map: dict[bytes, tuple[str, str]]
+) -> tuple[str, tuple[str, str] | None]:
+    mac = ":".join(f"{byte:02X}" for byte in reversed(ble_addr))
+
+    hostinfo = hostname_map.get(ble_addr)
+    if hostinfo is None:
```

Comment:
Do we need the `if` here?
(it seems that returning `(mac, hostinfo)` is enough...)